axiom-cloudwatch-lambda/handler.py

```diff
@@ -0,0 +1,135 @@
+import base64
+import gzip
+import json
+import logging
+import os
+import re
+
+level = os.getenv("log_level", "INFO")
+logging.basicConfig(level=level)
+logger = logging.getLogger()
+logger.setLevel(level)
+
+
+# Standard out from Lambdas.
+std_matcher = re.compile("\d\d\d\d-\d\d-\d\d\S+\s+(?P<request_id>\S+)")
```

Comment:
I don't get what is the use of the regexes and it is not used

Reply:
fixed